Build/Projects/MonoGame.Tests.definition

```diff
@@ -33,6 +33,33 @@
   </Properties>
 
   <Files>
+    <!-- Dependencies -->
+    <None Include="..\ThirdParty\Dependencies\SDL\Windows\x64\SDL2.dll">
+      <Platforms>WindowsGL</Platforms>
+      <Link>SDL2.dll</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="..\ThirdParty\Dependencies\openal-soft\Windows\x64\soft_oal.dll">
```

Comment:
Visual Studio already copies the libraries into the `x64`/`x86` folders thanks to the framework reference, but this doesn't hurt even then.